pkgs/by-name/sa/salut/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, pkg-config
+, fontconfig
+, libxkbcommon
+, wayland
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "salut";
+  version = "unstable-2022-12-17";
+
+  src = fetchFromGitLab {
+    owner = "snakedye";
+    repo = "salut";
+    rev = "aa57c4d190812908a9c32cd49cff14390c6dfdcb";
+    hash = "sha256-W0lhhImSXtYJDeMbxyEioYu/Bh7ZclwR1/5DzNbxM8o=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "msg-api-0.1.0" = "sha256-SGEr9kitvD+KZPGejwDAISK6ERk7G2uskxX8ljiJ2To=";
+      "smithay-client-toolkit-0.16.0" = "sha256-kiTO+BZIgpuwAr6gs9FCqz81jRg+3dV4NxzOX9kbJOc=";
+      "snui-0.1.4" = "sha256-jJL9ukSOczHjPM2EAXcXcz620SK4DQfr+xAT8v7fp9o=";
+      "snui-adwaita-0.1.0" = "sha256-pILhLMzqnhLZfGAXT8QQn6x+IvwG7CSa96wZqq1yrLY=";
+    };
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    fontconfig
+    libxkbcommon
+    wayland
+  ];
+
+  meta = {
+    description = "A sleek notification daemon for Wayland";
```

Comment:
```suggestion
    description = "Sleek notification daemon for Wayland";
```

Reply:
<https://nixos.org/manual/nixpkgs/unstable/#var-meta-description>

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
> 
> Wrong: "libpng is a library that allows you to decode PNG images."
>
> Right: ___"A library___ for decoding PNG images"

Reply:
I guess it depends on how you read it:
- `pname` is `meta.description`
- `pname` is a/an `meta.description`

Personally I prefer the latter, but I guess it was always intended to be the former: 79d3e0926d68bb760f1a9f28c862c38e6cd8cc45